packages/ui/components/landing/Header/MobileNavBar/MobileNavBar.tsx

```diff
@@ -24,6 +24,7 @@ const icons = {
   "/web/recruitment": <PlusOutlined />,
   "/web/team": <TeamOutlined />,
   "/web/hall-of-fame": <CrownOutlined />,
+  "/web/dojocon": <CrownOutlined />,
```

Comment:
Change this to another icon 🙏

Reply:
Thanks for the heads up! I was just waiting for Júlio to tell me his preferences :pray: